commands.pm

```diff
@@ -236,9 +236,9 @@ sub get_temp_file {
 sub run_daemon {
     my ($port, $isotovideo) = @_;
 
-    # allow up to 20GB - hdd images
-    $ENV{MOJO_MAX_MESSAGE_SIZE}   = 1024 * 1024 * 1024 * 20;
-    $ENV{MOJO_INACTIVITY_TIMEOUT} = 300;
+    # allow up to 20 GiB for uploads of big hdd images
+    $ENV{MOJO_MAX_MESSAGE_SIZE}   //= $bmwqemu::vars{FILE_UPLOAD_MAX_MESSAGE_SIZE}   // 1024 * 1024 * 1024 * 20;
```

Comment:
I guess the calculation `1024 * 1024 * 1024 * 20` can be simplified with the perl helper module we recently referenced in https://github.com/os-autoinst/os-autoinst/pull/1746.

Also I suggest to use a variable for the size in GB, e.g.

```suggestion
    $ENV{MOJO_MAX_MESSAGE_SIZE}   //= ($bmwqemu::vars{FILE_UPLOAD_MAX_MESSAGE_SIZE_GB} // 20) * 1024**3;
```

Reply:
I would rather keep it consistent with the upstream `MOJO_MAX_MESSAGE_SIZE` variable and not specify this in GiB.

Reply:
Fine, but *normally* HTTP limits are not in GiB.

Reply:
yeah but *normally* users would not exceed 20 GiB then ;)